pkg/test/step.go

```diff
@@ -451,7 +455,11 @@ func (s *Step) LoadYAML(file string) error {
 
 	for _, obj := range s.Apply {
 		if obj.GetObjectKind().GroupVersionKind().Kind == "TestStep" {
-			s.Step = obj.(*kudo.TestStep)
+			if testStep, ok := obj.(*kudo.TestStep); ok {
```

Comment:
The kind is tested in the line above

Reply:
Right, but that's orthogonal to the type, as the added test shows.